agenta-web/src/components/Evaluations/EvaluationErrorModal.tsx

```diff
@@ -1,15 +1,30 @@
-import React from "react"
+import React, {Dispatch, SetStateAction} from "react"
 import {Modal, Button} from "antd"
 
 interface Props {
     isModalOpen: boolean
-    onClose: () => void
     handleNavigate: () => void
     message: string
+    btnText: string
+    closeBtnText: Dispatch<SetStateAction<string>>
```

Comment:
Please avoid passing state setter functions as props here. They can be called from the parent component inside the `onClose` callback.